fvm/src/call_manager/default.rs

```diff
@@ -315,12 +328,26 @@ where
                     .map_err(Abort::Fatal)?;
 
                 // Invoke it.
-                let return_block_id = invoke.call(&mut store, (param_id,))?;
+                let res = invoke.call(&mut store, (param_id,));
+
+                // First, charge gas for the "latest" fuel use (all the fuel used since the most recent syscall)
+                let fuel_consumed = store
+                    .fuel_consumed()
+                    // TODO: Check whether this ever returns None (eg if 0 fuel has been consumed)
```

Comment:
This can only return `None` when fuel consumption is disabled in the configuration.

Reply:
Good, so long as we avoid doing that, we're good.